### PR TITLE
Add a section on Extensible Priorities

### PR DESCRIPTION
**Pull request by ianswett**
With an initial set of text.

### CODE REVIEW

File: draft-kazuho-httpbis-priority.md [hunk 1]
```diff
@@ -226,6 +226,20 @@ extensible priority scheme when and only when all the requests that are to be
...
 has negotiated the use of the extensible priority scheme (see {{fairness}}).
 
+# Extensible Priorities
+
+This priority design is intended to be extensible, and uses a sequence of
+key-value pairs encoded as a Structured Headers Dictionary
+({{!STRUCTURED-HEADERS}}) to enable extensibility. Each dictionary member
+represents a parameter of the Priority header field.
+
+Care must be taken to ensure any use of extisting parameters are either
```
Comment (LPardue) [inline draft-kazuho-httpbis-priority.md]:
```suggestion
Care must be taken to ensure any use of existing parameters are either
```

Comment (LPardue) [reply draft-kazuho-httpbis-priority.md]: I think the paragraph would also benefit from starting with the subject e.g. "When attempting to extend priorities, care must be taken.."

**Review by LPardue — CHANGES_REQUESTED** [review]

**Review by LPardue — APPROVED** [review]
This is a good starting point, I suggest we merge it as-is in order to permit further progress.

**Comment by kazuho** [comment]
Thank you for the text. I think that the clarification on extensibility is fine.

OTOH, this looks to me like an in-progress work of refactoring being merged.

The previous structure of the document was to first describe the header and the parameters, then state that the frame reuses that encoding.

This PR changed the structure to first describe that SH is used by headers and frame, then talk about the header field along with and parameters, then talk separately about frames.

I think we should either restore the original structure (requires minimal work), or go ahead and refactor the text so that we discuss parameters in a conveyer-independent way, then talk about headers and frames.

**Comment by LPardue** [comment]
I'll take the blame here. I think a few people would appreciate a document restructure in order to make changes that are more content full and accurate

**Comment by ianswett** [comment]
I was going to do the latter ("go ahead and refactor the text so that we discuss parameters in a conveyer-independent way, then talk about headers and frames.") this afternoon after outstanding PRs are merged.

**Comment by kazuho** [comment]
I share the view that restructuring is preferable.

**Pull request merged**